task/06-conditions-n-loops-tasks.js

```diff
@@ -82,7 +101,10 @@ function getSumBetweenNumbers(n1, n2) {
  *   10,10,10 =>  true
  */
 function isTriangle(a,b,c) {
-    throw new Error('Not implemented');
+     
+    if(a+b>c && a+c>b && b+c>a)
+        return true;
+    return false;
```

Comment:
Just `return (a+b>c && a+c>b && b+c>a)` with the same result

Please, see https://javascript.info/logical-operators